Resources/translations/fr/content_type.fr.xlf

```diff
@@ -0,0 +1,147 @@
+    <body>
+      <trans-unit id="4f1cfd17c4ea63fdcfb4a10fda0755ff" resname="content_type.dashboard_title">
+        <source>content_type.dashboard_title</source>
+        <target>Types de Contenu</target>
+      </trans-unit>
+      <trans-unit id="80d8a14d1f4ec432a49b7acf8af8f3d2" resname="content_type.group.list">
+        <source>content_type.group.list</source>
+        <target>Groupes de type de contenu</target>
+      </trans-unit>
+      <trans-unit id="3dc2dbac68487a7d3f87f767fc4bbd2e" resname="content_type.group.name">
+        <source>content_type.group.name</source>
+        <target>Nom</target>
+      </trans-unit>
+      <trans-unit id="7415e532c85215d11540f3783967446d" resname="content_type.group.id">
+        <source>content_type.group.id</source>
+        <target>ID</target>
+      </trans-unit>
+      <trans-unit id="a5e8ac6ece0bfd015d0fff25748474cb" resname="content_type.group.edit">
+        <source>content_type.group.edit</source>
+        <target>Éditer</target>
+      </trans-unit>
+      <trans-unit id="2cf33e780bfbad13ad16fcf5634ec70c">
+        <source>content_type.group.new</source>
+        <target>Créer un groupe de type de contenu</target>
+      </trans-unit>
+      <trans-unit id="7b24488c5d8ce01a3dd1a49ca7381d0b" resname="content_type.group">
+        <source>content_type.group</source>
+        <target>Groupe de type de contenu</target>
+      </trans-unit>
+      <trans-unit id="d95b927203e077ad6f20662ae465821d">
+        <source>content_type.group.create.title</source>
+        <target>Créer un nouveau groupe de type de contenu</target>
+      </trans-unit>
+      <trans-unit id="80644ee5d10e849935daf17deddf5e6f">
+        <source>content_type.group.edit.title</source>
+        <target>Éditer le groupe de type de contenu "%identifier%"</target>
+      </trans-unit>
+      <trans-unit id="7a70af7dfe97699edb120836224eee1e">
+        <source>content_type.group.cancel</source>
+        <target>Annuler</target>
+      </trans-unit>
+      <trans-unit id="a8d0b6d82750581100743046985fcf80">
+        <source>content_type.group.notification.created</source>
+        <target>Le groupe de type de contenu "%identifier%" a été correctement créé.</target>
+      </trans-unit>
+      <trans-unit id="c20c719ac1ac6b400867977af4a0c6e1">
+        <source>content_type.group.notification.updated</source>
+        <target>Le groupe de type de contenu "%identifier%" a été correctement mis à jour.</target>
+      </trans-unit>
+      <trans-unit id="20d385d52df00a6a1aa4d01c7e75f4e5">
+        <source>content_type.group.deleted</source>
+        <target>Le groupe de type de contenu "%identifier%" a été correctement supprimé.</target>
+      </trans-unit>
+      <trans-unit id="147661b50276b00ffac1bb715bdb4746">
+        <source>content_type.group.cannot_delete.has_content_types</source>
+        <target>Impossible de supprimer le groupe de contenu "%identifier%" car il contient toujours des types de contenu.</target>
```

Comment:
I guess you meant *le groupe de type de contenu* but to make the sentence a bit lighter, I would write: *Impossible de supprimer le groupe "%identifier%" car il contient toujours des types de contenu.*